client/src/index.jsx

```diff
@@ -2,8 +2,8 @@ import { StrictMode } from 'react';
 import { render } from 'react-dom';
 import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
-import App from 'components/app';
-import store from 'store/store';
+import { Routing as App } from 'components/app';
```

Comment:
```suggestion
import { App } from 'components/app';
```
should be the same as folder name